sdk/database/dbplugin/database.proto

```diff
@@ -86,6 +107,8 @@ service Database {
 	rpc RotateRootCredentials(RotateRootCredentialsRequest) returns (RotateRootCredentialsResponse);
 	rpc Init(InitRequest) returns (InitResponse);
 	rpc Close(Empty) returns (Empty);
+  rpc SetCredentials(SetCredentialsRequest) returns (SetCredentialsResponse);
+  rpc GenerateCredentials(Empty) returns (GenerateCredentialsResponse);
```

Comment:
Can you replace spaces with tabs in this file?

Reply:
Done in a6affaf4b